yarn.lock

```diff
@@ -3198,18 +3198,7 @@ __metadata:
   languageName: unknown
   linkType: soft
 
-"@department-of-veterans-affairs/css-library@npm:^0.11.1":
-  version: 0.11.1
-  resolution: "@department-of-veterans-affairs/css-library@npm:0.11.1"
-  dependencies:
-    "@divriots/style-dictionary-to-figma": "npm:^0.4.0"
-    "@uswds/uswds": "npm:^3.8.1"
-    rimraf: "npm:^5.0.5"
-  checksum: 10/49d5685e4bced6cdb91e8cc20f133944a70cebfb3890c24a9ffa98e63e15417c17eea47bb2ce87266626df23a889ea2a185ffd064a06764218cdaf7b2d66386b
-  languageName: node
-  linkType: hard
-
-"@department-of-veterans-affairs/css-library@npm:^0.12.2, @department-of-veterans-affairs/css-library@workspace:packages/css-library":
+"@department-of-veterans-affairs/css-library@npm:^0.12.3, @department-of-veterans-affairs/css-library@workspace:packages/css-library":
```

Comment:
I needed to link the CSS Library with the other component-library packages in order to get the testing preview pages to work in Chromatic but I'm going to revert this stuff before merging too after testing is done.